code/modules/mob/living/carbon/carbon.dm

```diff
@@ -745,9 +745,13 @@
 		. += INFINITY
 
 // permeability: now slightly more sane and probably functional!
-/mob/living/carbon/get_permeability(def_zone)
+/mob/living/carbon/get_permeability(def_zone, linear = FALSE)
```

Comment:
This ``linear`` arg isn't in the other cases of ``get_permeability`` which will cause problems in the future.